### DISCUSSION TITLE
New Field Options

### DISCUSSION THREAD

**Discussion by m26dvd** [robbrad/UKBinCollectionData#998] [Ideas]
When creating the config to allow both Babergh District Council and Mid Suffolk District Council, it became apparent that both of these councils do not provide schedules based on individual addresses and as such, only provide PDFs with Week based collections

In order to create the config, it meant I re-used the Postcode and House Number fields to ask the user for their "Collection Day" and "Collection Week" which is confusing for users if they don't read the wiki that gives them this instruction.

It would be good if we could have additional fields added to the schema that we can use to capture and validate this data rather than use fields designated for postcode and house number. My belief is that these only show on the GUI when the underlying council schema is configured as such?

Field Name suggestions:
collection_week
collection_day

FYI @robbrad @dp247

**Comment by dp247** [comment]
I'd quite like a "custom field" - seem to remember thinking something similar when we were passing UPRNs into postcodes 😂.

Collection day/week make sense, but could just be as easy as "user data" passed in a list (although knowing nothing of HA's config, is that even possible lol?)
